crates/newtype-uuid/LICENSE-APACHE

```diff
@@ -0,0 +1 @@
+../../LICENSE-APACHE
```

Comment:
(asking because i'm curious) does `cargo publish` behave nicely with symlinks?

Reply:
Yeah (a contributor added this to nextest and I had the same q then -- works great)